pkg/operations/start.go

```diff
@@ -80,6 +81,17 @@ func StartVM(vm *api.VM, debug bool) error {
 		PortBindings: vm.Spec.Network.Ports, // Add the port mappings to Docker
 	}
 
+	// Only set envVars if at least one was configured
+	var envVars []string
+	parts := strings.Split(vm.GetAnnotation((constants.IGNITE_ENV_VAR_ANNOTATION)), ",")
+	for _, part := range parts {
+		if part == "" {
+			continue
+		}
+		envVars = append(envVars, part)
+	}
+	config.EnvVars = envVars
```

Comment:
It's okay to remove this section since the VM Config file is already a sufficient mechanism to pass the config to ignite-spawn.